.github/workflows/pynetbox_codecov.yaml

```diff
@@ -1,44 +1,40 @@
-name: Pynetbox Data Uploader Tests
-
-on:
-  push:
-    branches:
-      - master
-  pull_request:
-    paths:
-      - "pynetbox_data_uploader/**"
-      - ".github/workflows/pynetbox.yaml"
-
-jobs:
-  lint_test_codecov:
-    runs-on: ubuntu-20.04
-    strategy:
-      matrix:
-        python-version: ["3.8", "3.x"]
-    steps:
-      - uses: actions/checkout@v3
-      - name: Set up Python ${{ matrix.python-version }}
-        uses: actions/setup-python@v3
-        with:
-          python-version: ${{ matrix.python-version }}
-          cache: "pip"
-
-      - name: Install dependencies
-        run: |
-          python -m pip install --upgrade pip
-          cd pynetbox_data_uploader
-          pip install -r requirements.txt
-          
-
-      - name: Analyse with pylint
-        run: cd pynetbox_data_uploader && pylint . --rcfile=.pylintrc
-
-      - name: Run tests and collect coverage
-        run: cd pynetbox_data_uploader && python3 -m pytest --cov-report xml:coverage.xml --cov
-
-      - name: Upload coverage to Codecov
-        uses: codecov/codecov-action@v3
-        with:
-          token: ${{secrets.CODECOV_TOKEN}}
-          files: ./pynetbox_data_uploader/coverage.xml
-
+name: Pynetbox Codecov
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    paths:
+      - "pynetbox_data_uploader/**"
+      - ".github/workflows/pynetbox.yaml"
+
+jobs:
+  Codecov:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version: ["3.x"]
```

Comment:
I guess you removed python 3.8 compatibility because you're using python switch statements - are you sure we don't need python3.8 compatibility when running this?

Reply:
I'm not sure. I have removed python 3.8 for now because it will always fail with the switch statements.